src/sortStudents.ts

```diff
@@ -1,16 +1,55 @@
+// import { valueToNode } from "@babel/types";
 
 export interface Student {
   // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
   // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+const getAverage = (value : number[]): number => value
+  .reduce((a, b) => a + b, 0) / value.length;
```

Comment:
Better use more meaningful variables, for example.
```suggestion
const getAverage = (grades : number[]): number => grades
  .reduce((sum, grade) => sum + grade, 0) / value.length;
```
or this way if you want to reuse this function in other project 👇 
```suggestion
const getAverage = (dataArray : number[]): number => dataArray
  .reduce((sum, current) => sum + current, 0) / value.length;
```